### PR TITLE
fix(ng-update): migrations not running for release candidate versions

### PR DESCRIPTION
**Pull request by devversion**
7a433f6e5ba1b7c0aa0fd0f05628eb882de21dfa set up the migrations for v9, but we actually forgot
that we want to run migrations in the next and RC versions too.

### CODE REVIEW

**Comment by devversion** [comment]
I pushed a second commit that validates the migration collection in the future.

File: tools/release/release-output/output-validations.ts [hunk 1]
```diff
@@ -126,8 +128,47 @@ export function checkMaterialPackage(packagePath: string): string[] {
  */
 export function checkCdkPackage(packagePath: string): string[] {
   const prebuiltFiles = glob('*-prebuilt.css', {cwd: packagePath}).map(path => basename(path));
-
   return ['overlay', 'a11y', 'text-field']
-      .filter(name => !prebuiltFiles.includes(`${name}-prebuilt.css`))
-      .map(name => `Could not find the prebuilt ${name} styles.`);
+                       .filter(name => !prebuiltFiles.includes(`${name}-prebuilt.css`))
```
Comment (mmalerba) [inline tools/release/release-output/output-validations.ts]: nit: indentation got weird for some reason

File: tools/release/release-output/release-packages.ts [hunk 2]
```diff
@@ -8,3 +8,8 @@ export const releasePackages = [
   'material-experimental',
   'material-moment-adapter',
 ];
+
+export const packagesWithMigrations = [
```
Comment (mmalerba) [inline tools/release/release-output/release-packages.ts]: can we just check `packageJson['ng-update']` to determine this automatically without needing to maintain a separate list?

File: tools/release/release-output/output-validations.ts (hunk 1 above)
Comment (devversion) [reply tools/release/release-output/output-validations.ts]: This is actually what clang-format did when I wrapped the expression for testing in an array.

File: tools/release/release-output/release-packages.ts (hunk 2 above)
Comment (devversion) [reply tools/release/release-output/release-packages.ts]: Done.

**Comment by sawyerbutton** [comment]
Any idea about how material fix  

> Increased main.js bundle size but less lazy chunks after updating to angular 9.0.0-rc.1 #16146
 
problem?

**Pull request merged**